orfs

```diff
@@ -15,6 +15,9 @@ fi
 
 export WORK_HOME=$BUILD_DIR/$RULEDIR
 
+# Make bazel-bin writable
+chmod -R +w $WORK_HOME
```

Comment:
Hmm... OK for now, but it seems to me that this should be in the _make scripts. Are we hiding a more subtle problem?